viam_cartographer.go

```diff
@@ -120,12 +120,12 @@ func TerminateCartoLib() error {
 func initSensorProcesses(cancelCtx context.Context, cartoSvc *CartographerService) {
 	spConfig := sensorprocess.Config{
 		CartoFacade:              cartoSvc.cartofacade,
-		Lidar:                    cartoSvc.lidar.timed,
-		LidarName:                cartoSvc.lidar.name,
-		LidarDataFrequencyHz:     cartoSvc.lidar.dataFrequencyHz,
-		IMU:                      cartoSvc.imu.timed,
-		IMUName:                  cartoSvc.imu.name,
-		IMUDataFrequencyHz:       cartoSvc.imu.dataFrequencyHz,
+		Lidar:                    cartoSvc.lidar,
+		LidarName:                cartoSvc.lidar.Name(),
+		LidarDataFrequencyHz:     cartoSvc.lidar.DataFrequencyHz(),
+		IMU:                      cartoSvc.imu,
```

Comment:
Can we just pass in imu? in one param as opposed to having 3?

Reply:
Yes, doing it already in the next PR :)